cli/app.php

```diff
@@ -175,15 +175,26 @@ function (ConsoleCommandEvent $event) {
     /**
      * Register a symbolic link with Valet.
      */
-    $app->command('link [name] [--secure]', function (OutputInterface $output, $name, $secure) {
+    $app->command('link [name] [--secure] [--isolate]', function (OutputInterface $output, $name, $secure, $isolate) {
         $linkPath = Site::link(getcwd(), $name = $name ?: basename(getcwd()));
 
         info('A ['.$name.'] symbolic link has been created in ['.$linkPath.'].');
 
         if ($secure) {
-            $this->runCommand('secure '.$name);
+            $this->runCommand('secure');
```

Comment:
What's the benefit of dropping `$name` here?

Reply:
It's not passed to the underlying command in any way